doc/fonts_quickref.md

```
@@ -0,0 +1,222 @@
+```go
+sizes := []float64{8, 10, 12, 14, 16, 18, 20, 24}
+for _, size := range sizes {
+    pdf.SetFont("Sarabun", "", size)
+    pdf.Cell(0, size/2+2, fmt.Sprintf("Size %.0f", size))
+    pdf.Ln(size/2 + 4)
+}
+```
+
+## File Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)